package.json

```diff
@@ -1,126 +1,126 @@
 {
-    "name": "sulu-sulu",
-    "license": "MIT",
-    "repository": "https://github.com/sulu/sulu.git",
-    "scripts": {
-        "preinstall": "node preinstall.js",
-        "lint:js": "eslint . --cache",
-        "lint:scss": "stylelint src/Sulu/Bundle/*/Resources/js/**/*.scss",
-        "flow": "flow",
-        "test": "jest",
-        "depcruise": "depcruise src/Sulu/Bundle/*/Resources/js -c dependency-cruiser.json",
-        "styleguide": "styleguidist server",
-        "styleguide:build": "styleguidist build",
-        "build": "webpack --mode production",
-        "watch": "webpack --mode development --watch"
+  "name": "sulu-sulu",
```

Comment:
@b3h3m0th please undo this changes of this file.

Reply:
Sorry about that 😬